packages/composer-rest-server/test/system.js

```
@@ -340,44 +340,44 @@ describe('System REST API unit tests', () => {
-                    res.should.be.json;
-                    res.body.filter((tx) => {
-                        return tx.$class === 'org.acme.bond.PublishBond';
-                    }).sort((a, b) => {
-                        return a.ISINCode.localeCompare(b.ISINCode);
-                    }).map((tx) => {
-                        delete tx.transactionId;
-                        delete tx.timestamp;
-                        return tx;
-                    }).should.deep.equal(transactionData);
+                    // res.should.be.json;
+                    // res.body.filter((tx) => {
+                    //     return tx.$class === 'org.acme.bond.PublishBond';
+                    // }).sort((a, b) => {
+                    //     return a.ISINCode.localeCompare(b.ISINCode);
+                    // }).map((tx) => {
+                    //     delete tx.transactionId;
+                    //     delete tx.timestamp;
+                    //     return tx;
+                    // }).should.deep.equal(transactionData);
                 });
         });
 
     });
 
-    describe('GET /transactions/:id', () => {
+    describe('GET /historian/:id', () => {
 
         it('should return the specified transaction', () => {
             return chai.request(app)
-                .get('/api/system/transactions/' + transactionIds[0])
+                .get('/api/system/historian/' + transactionIds[0])
                 .then((res) => {
-                    res.should.be.json;
-                    const tx = res.body;
-                    delete tx.transactionId;
-                    delete tx.timestamp;
-                    tx.should.deep.equal(transactionData[0]);
-                });
+                    // res.should.be.json;
+                    // const tx = res.body;
+                    // delete tx.transactionId;
+                    // delete tx.timestamp;
+                    // tx.should.deep.equal(transactionData[0]);
+                }).catch((err)=>{});
```

Needs asserts and removing of the empty catch block!